src/components/basic-element/types/link.jsx

```diff
@@ -26,6 +26,11 @@ var spec = new Spec('link', assign({
     validation: React.PropTypes.string,
     default: ''
   },
+  targetWebURL: {
+    category: 'attributes',
+    validation: React.PropTypes.string,
```

Comment:
Would there be any benefit of extending this into a url validation?

(I'm guessing it might be more work than it's worth at this point though)